docs/conf.py

```diff
@@ -153,6 +153,9 @@
 # Disable google style docstrings
 napoleon_google_docstring = False
 
+# Disable the use of param, which prevents a distinct "Other Parameters" section
+napoleon_use_param = False
```

Comment:
Does this have any effect on how the docs are rendered?

Reply:
Seemingly not.

Reply:
As I understand it, `napoleon_use_param=False` bakes in the usual formatting for the parameter lists, while `napoleon_use_param=True` uses the reStructuredText directive `:param:`, which allows for a theme to specify a non-default formatting.  Since our docs just use the default formatting of parameter lists, there is no difference in the rendered output.